packages/main/src/components/AnalyticalTable/VerticalResizer.tsx

```diff
@@ -156,6 +162,14 @@ export const VerticalResizer = (props: VerticalResizerProps) => {
     };
   }, []);
 
+  const isInitial = useRef(true);
+  useEffect(() => {
+    if (!isInitial.current && rowsLength <= visibleRows) {
+      handleOnLoadMore({ type: 'custom' } as Event);
```

Comment:
`type: custom` sounds a bit generic to me, but I also have no idea on how to name it better. `tableResize` would be nice but I think we can't distinguish here whether the event was really fired due to a resize...